test/extended/testdata/verifyservice-pipeline-template.yaml

```diff
@@ -0,0 +1,114 @@
+apiVersion: v1
+kind: Template
+labels:
+  template: nodejs-example
+metadata:
+  annotations:
+    iconClass: icon-jenkins
+    tags: instant-app,jenkins
+  name: nodejs-example
+objects:
+- apiVersion: v1
+  kind: Service
+  metadata:
+    annotations:
+      description: Normal Service to be tested
+    name: nodejs-example
+  spec:
+    ports:
+    - name: web
+      port: 8080
+      targetPort: 8080
+    selector:
+      name: nodejs-example
+- apiVersion: v1
+  kind: Service
+  metadata:
+    annotations:
+      description: Headless Service to be tested
+    name: nodejs-example-headless
+  spec:
+    clusterIP: None
+    ports:
+    - port: 8080
+      targetPort: 8080
+    selector:
+      name: nodejs-example
+- apiVersion: v1
+  kind: DeploymentConfig
+  metadata:
+    annotations:
```

Comment:
annotation is unnecessary ...let's remove

Reply:
Removed all annotations